client/state/data-layer/wpcom/sites/jitm/index.js

```diff
@@ -66,38 +66,33 @@ export const doFetchJITM = ( action ) =>
 	http(
 		{
 			method: 'GET',
-			path: `/jetpack-blogs/${ action.siteId }/rest-api/`,
+			apiNamespace: 'wpcom/v3',
+			path: `/sites/${ action.siteId }/jitm`,
 			query: {
-				path: '/jetpack/v4/jitm',
-				query: JSON.stringify( {
-					message_path: action.messagePath,
-					query: action.searchQuery,
-				} ),
-				http_envelope: 1,
+				message_path: action.messagePath,
+				query: action.searchQuery,
 				locale: action.locale,
```

Comment:
We might want to double check if this `locale` is necessary/used for anything. I don't see it declared specifically in the `WPCOM_REST_API_V3_Endpoint_JITM` endpoint, so it could potentially be something we might want to cleanup:

https://github.com/Automattic/jetpack/blob/45a208ffdb312ed27c74b36a4687aeceb785e7c5/projects/plugins/jetpack/_inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v3-endpoint-jitm.php

Reply:
Agreed. I can't see it in the `jetpack/v4` either. If we can confirm [this](https://github.com/Automattic/wp-calypso/pull/101783#discussion_r2011606029) then I think we can safely remove it